zeppelin-client-examples/src/main/java/org/apache/zeppelin/client/examples/ZeppelinClientExample.java

```diff
@@ -41,6 +41,9 @@ public static void main(String[] args) throws Exception {
       noteId = zClient.createNote(notePath);
       System.out.println("Created note: " + noteId);
 
+      zClient.renameNote(noteId, notePath + "_rename");
+      System.out.println("Rename note: " + noteId + " to " + notePath + "_rename ");
```

Comment:
Use getNote to verify the rename operation is successful

Reply:
Thank you for your review, I made som changes on 76e4da4